README.md

```diff
@@ -8,75 +8,93 @@
   <a href="https://www.npmjs.com/package/@algolia/netlify-plugin-crawler"><img src="https://img.shields.io/npm/v/algolia/netlify-plugin-crawler" alt="Version"></a>
 </p>
 
-<h2 align="center">Algolia Netlify integration</h2>
+<h2 align="center">Algolia Netlify plugin</h2>
```

Comment:
On this one, it is an integration on the same level than Magento or Shopify might be, I think it's important we keep the wording.
This integration itself uses a Netlify plugin to trigger the crawl, the actual crawler, and a frontend library which uses this bundle.

Reply:
No problem, I made this edit because the documentation says "plugin" everywhere. Should we update it?

Reply:
I'm not sure everyone agrees with what I just said, which probably explains the inconsitency.
@bodinsamuel You're the main author of the doc, WDYT?

Reply:
There is 2 world colliding here, netlify is calling that plugin and we call it integration. 
we need to pick, I don't think there is a good answer

Reply:
IMO we should go with plugin. We depend on Netlify here, and they use the term "plugin" everywhere so we should stick to it. We can always file it under "Integrations" when we add it to the main docs, but we don't want to confuse Netlify users.

Reply:
okay if you feel like it's not an issue to have different wording let's go for "plugin"

Reply:
That's a valid point. The fact that the plugin is only a sub-part of this repo might be confusing for people who want to understand the tooling, but that's indeed less important than the end users.